### DISCUSSION TITLE
OMG DDS-TSN Standard has been published, will eProsima implement it？

### DISCUSSION THREAD

**Discussion by wddpfx** [eProsima/Fast-DDS#4611] [General]
https://www.omg.org/spec/DDS-TSN/

**Comment by JesusPoderoso** [comment]
Hi @wddpfx, the OMG DDS-TSN standard is still a beta.
We definitely have it in our roadmap but it is not our top priority for now.
If you are interested in it, consider contacting our support team via email: support@eProsima.com

**Reply by wddpfx** [reply]
OK。
Can you answer me others question?
this one : https://github.com/eProsima/Fast-DDS/discussions/4504
and this one :https://github.com/eProsima/Fast-DDS/discussions/4505

**Comment by wddpfx** [comment]
好的，收到了你的邮件，我会尽快查看。
